src/domains/transaction/pages/SendTransfer/FormStep.tsx

```diff
@@ -135,10 +134,10 @@ export const FormStep = ({
 					<FormLabel label={t("COMMON.CRYPTOASSET")} />
 					<SelectNetwork
```

Comment:
We should use the `SelectNetworkDropdown` component here directly, to avoid the hack with the three networks on L138,

```
<SelectNetworkDropdown
    profile={profile}
    networks={[network]}
    selectedNetwork={network}
    isDisabled
/>
```

Reply:
@dated indeed. Fixed. Thanks!